merkle/sparse_merkle_tree.go

```diff
@@ -442,12 +478,16 @@ func (s SparseMerkleTreeReader) BatchInclusionProof(ctx context.Context, rev int
 		}
 		r[string(index)] = ri
 	}
+	span.End()
```

Comment:
Might be Github diffs confusing things but does that match the span that's had the End() already deferred?

Reply:
should be the one at L456, no?
(I can rename them if you prefer?

Reply:
No I see it now. I think renaming just adds noise.